### DISCUSSION TITLE
The application safe_pose_changer has stopped unexpectedly

### DISCUSSION THREAD

**Discussion by Kai-X-Org** [HumanSupportRobot/QandA#72] [Q&A]
Upon start up of the simulator, with `roslaunch hsrb_gazebo_launch hsrb_megaweb2015_world.launch,` I get the message from Ubuntu saying that the application safe_pose_changer has stopped unexpectedly. What functions does this impact? And what is the fix? Thanks a lot!

**Comment by masudam** [comment]
I tried safe_pose_changer in my own environment, and it worked without any issues. Therefore, providing additional information might be helpful in resolving the problem.
 ex) 

- error logs

- other ROS-related programs that were running (or were launched just before the issue)

- version information of the deb packages (If you are using safe_pose_changer, the deb packages related to tmc_manipulation_** are likely relevant, you can check with dpkg -l | grep tmc_manipulation)
